labelbox/schema/data_row_metadata.py

```diff
@@ -480,6 +483,7 @@ def bulk_upsert(
 
 
         >>> metadata = DataRowMetadata(
+        >>>                 global_key="global-key",
```

Comment:
Can we specify in the docstring here that users should supply either global_key or data_row, and show usage of only one of them in the example? I feel like it's confusing to show us setting both in the example
```
        """Upsert metadata to a list of data rows.

        You may specify data row by either global_key or data_row_id

        >>> metadata = DataRowMetadata(
        >>>                 global_key="global-key", # Alternatively, set data_row_id="data-row-id"
        >>>                 fields=[
        >>>                        DataRowMetadataField(schema_id="schema-id", value="my-message"),
        >>>                        ...
        >>>                    ]
        >>>    )
        >>> mdo.batch_upsert([metadata])
```